src/include/storage/index/index_builder.h

```diff
@@ -23,6 +24,7 @@ class IndexBuilder {
   catalog::index_oid_t index_oid_{0};
   ConstraintType constraint_type_ = ConstraintType::INVALID;
   catalog::IndexSchema key_schema_;
+  bool ordered_ = true;
```

Comment:
Sure. Do we want it in IndexSchema (and eventually persisted in the catalog), or just an argument set on the IndexBuilder?

Reply:
It probably should be in IndexSchema with the goal of eventual persistence (`pg_index`?) since we would want to ensure replicas stay in sync with the master.